src/controller/SetUpCodePairer.cpp

```diff
@@ -243,6 +253,39 @@ CHIP_ERROR SetUpCodePairer::StopConnectOverSoftAP()
     return CHIP_NO_ERROR;
 }
 
+#if CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF
+static std::shared_ptr<SetUpCodePairer> SetUpCodeParserPtr;
```

Comment:
Again, I don't think this works.  SetUpCodePairer lifetime is not managed via the shared ptr....